LibraBFT/Impl/Properties/VotesOnceDirect.agda

```diff
@@ -39,29 +39,29 @@ module LibraBFT.Impl.Properties.VotesOnceDirect where
                    → Meta-Honest-PK pk → (sig : WithVerSig pk v)
                    → MsgWithSig∈ pk (ver-signature sig) (msgPool pre)
                    → ValidSenderForPK (availEpochs pre) v pid pk
-                   → (₋epEC (peerStates pre pid)) ^∙ epEpoch ≡ (v ^∙ vEpoch)
-                   → v ^∙ vRound ≤ (₋epEC (peerStates pre pid)) ^∙ epLastVotedRound
-  oldVoteRound≤lvr (step-s r (step-epoch _)) pidIn pkH sig msv vspk ep≡ = {!!}
+                   → (₋rmEC (peerStates pre pid)) ^∙ rmEpoch ≡ (v ^∙ vEpoch)
+                   → v ^∙ vRound ≤ (₋rmEC (peerStates pre pid)) ^∙ rmLastVotedRound
+  oldVoteRound≤lvr (step-s r (step-epoch _)) pidIn pkH sig msv vspk rm≡ = {!!}
   oldVoteRound≤lvr {pid = pid'} {pre = pre} (step-s r (step-peer {pid = pid} cheat@(step-cheat f c)))
-                    pidIn pkH sig msv vspk ep≡
```

Comment:
See other comments about `ValidSenderForPK⇒ep≡`.  Please change all of these `rm≡`s to `eid≡`.

Reply:
DONE